cmd/qat_plugin/kerneldrv/kerneldrv.go

```diff
@@ -153,6 +159,12 @@ func (dp *DevicePlugin) getOnlineDevices(iommuOn bool) ([]device, error) {
 			continue
 		}
 
+		// Ignore devices which are on the denylist.
+		if _, ok := devicesDenyList[matches[1]]; ok {
```

Comment:
Will this crash if `len(matches) == 1`?

Reply:
not sure to craft such line that the regexp would give that. `len() == 0` is checked but perhaps that should be `len() != 6`

Reply:
Yes, better to require the length to be something, rather than check all the cases which it can't be.

Reply:
changed to `len() != 6`